android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormActivity.java

```diff
@@ -270,79 +301,169 @@ public void refreshHiddenViews(boolean popup) {
                 if (viewData.has(JsonFormConstants.HIDDEN) && viewData.getBoolean(JsonFormConstants.HIDDEN)) {
                     toggleViewVisibility(curView, false, popup);
                 }
-            } catch (Exception e) {
+            } catch (JSONException e) {
                 Timber.e(e);
             }
         }
     }
 
     @Override
-    public void refreshSkipLogic(String parentKey, String childKey, boolean popup) {
+    public void refreshSkipLogic(String parentKey, String childKey, boolean popup, String stepName, boolean isForNextStep) {
```

Comment:
This is a public method. Changing method signature is a possible breaking change.

Is this a major version change? If so, please ignore all such comments.